app/assets/javascripts/discourse/app/components/user-menu/menu.hbs

```diff
@@ -30,6 +30,12 @@
             />
           {{/each}}
         </div>
+
+        <PluginOutlet
+          @name="user-menu-tabs-after"
+          @connectorTagName="span"
+          @outletArgs={{hash plugin=@plugin}}
```

Comment:
This menu does not have access to the `plugin` here. This should be removed.

A possible helpful argument to add here would probably be user. You can do this like:

`@outletArgs={{hash user=this.currentUser}}`

Reply:
great, will do ty